src/cljs/rems/spa.cljs

```diff
@@ -93,6 +94,11 @@
  (fn [db _]
    (:theme db)))
 
+(rf/reg-sub
+ :user-language
+ (fn [db _]
+   (:language (:user-settings db))))
+
```

Comment:
We should rather use the existing sub `:language` (from `user_settings.cljs`). This will not work.

Reply:
What do you mean will not work? it works in the UI

Reply:
If you look at the definition of `:language` subscription it is different to this definition and it is for a reason (it needs to e.g. check cookies and such).

Reply:
I can only find subscribtion for  ``` :language ```

Reply:
https://github.com/CSCfi/rems/blob/master/src/cljs/rems/user_settings.cljs#L28

Reply:
I guess we are done to one problem at this point. May I just ask about the language thing a bit more? @opqdonut @Macroz  isnt  ```(:user-settings db)``` supposed to be synchronized with the settings?

Reply:
Yes but that is not the same as what the chosen language is.

Reply:
You can check what the existing subscription does and see what it needs to check.

Reply:
now that this part has been tackled, I am kindly asking @Macroz to take a look again

Reply:
Sure I'll have a look